### PR TITLE
import for psatooltip updated

### PR DESCRIPTION
**Pull request by Diksha-Maurya**

### CODE REVIEW

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.151.67.206:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.151.67.206:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

File: src/pages/CropSelector/CropCalendarView/RenderCrops.jsx [hunk 1]
```diff
@@ -21,13 +21,6 @@ import '../../../styles/cropCalendarViewComponent.scss';
...
 import { setSaveHistory } from '../../../reduxStore/userSlice';
-import PSATooltip from '../../../components/PSAComponents/PSATooltip';
-
-const tooltipContent = () => (
-  <AcUnit
-    sx={{ color: 'white', backgroundColor: '#80D0FF', borderRadius: '5px' }}
-  />
-);
 
 const tooltipRatingContent = (crop, goal, councilShorthandRedux) => (
```
Comment (YazidZidane) [inline src/pages/CropSelector/CropCalendarView/RenderCrops.jsx:25]: There are a few `tooltipContent` left (including a few renamed to other like this), can you put all of them into the tooltip? You can use search in vscode for `tooltipContent` for all of them.

**Comment by mikahpinegar** [comment]
Your review app has been deployed and is available at 'http://57.151.67.206:80'.<br><br>[Learn More](https://aka.ms/testwithreviewapps) about how to test and provide feedback for the app.

**Review by YazidZidane — APPROVED** [review]

**Review by Adamws33 — APPROVED** [review]

**Pull request merged**